lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Task/TaskStatus.cs

```diff
@@ -289,15 +287,6 @@ public TaskStatusProto ToProto()
             }
         }
 
-        private byte[] SerializeException(Exception ex)
```

Comment:
So the change basically is to refactor the code to put SerializeToBinaryFormat() into utilities, right?